### DISCUSSION TITLE
Wix v4 - "Uninstall" in Wix4 is Flagged by Anti-Virus/Malwarelike Norton 360.

### DISCUSSION THREAD

**Discussion by garevaul** [wixtoolset/issues#7352] [Questions] [answered]
Hi,

"Uninstall" in Wix4 is Flagged by Anti-Virus/Malwarelike Norton 360.

This never happened in Wix v3.

Please Note:  "Installation" happens without any issue.

This is the Sequence and Screenshots

Regards

![image](https://user-images.githubusercontent.com/15125247/229502090-2f35826d-b5d7-4b1a-8d10-b51ee89d09db.png)

![image](https://user-images.githubusercontent.com/15125247/229502143-c98833b7-d5a3-43d2-8721-d3317d353411.png)

![Wix4NortonFlagDetails](https://user-images.githubusercontent.com/15125247/229530065-bbae76ed-e0fe-4a21-b421-c512c69218c3.png)

**Comment by robmen** [comment, marked as answer]
Meh, anti-virus software vendors are fickle monsters. Sign your content. If still a problem, go to the anti-virus vendor's website and send your content to be put on their safe-list. Works every time (as long as you aren't shipping a virus).

**Reply by garevaul** [reply]
Thanx @robmen. You are right about signing the installer. Now the Anti-Virus Does Not flag it.
